diff --git a/dist/cjs/src/fetch.js b/dist/cjs/src/fetch.js
index 030dcbb4adf87aeeb4fc8288f572c27e2c38ead0..c6bdc3384c954fa0ea7d3ec721f3c69d020d4050 100644
--- a/dist/cjs/src/fetch.js
+++ b/dist/cjs/src/fetch.js
@@ -39,7 +39,7 @@ exports.fetchWithRetry = fetchWithRetry;
 function fetchResource(fetch, url) {
     return fetchWithRetry(fetch, url).then((response) => response.text());
 }
-const PREFIX = 'https://raw.githubusercontent.com/cliqz-oss/adblocker/master/packages/adblocker/assets';
+const PREFIX = 'https://static.ziying.site/adblocker';
 exports.adsLists = [
     `${PREFIX}/easylist/easylist.txt`,
     `${PREFIX}/easylist/easylistgermany.txt`,
