diff --git a/dist/cjs/src/fetch.js b/dist/cjs/src/fetch.js
index 1be3fec374c8c367422559442ba5543e4d2b8d1c..3be9f4a6d13470fca6304cfab924d3fc7649a69b 100644
--- a/dist/cjs/src/fetch.js
+++ b/dist/cjs/src/fetch.js
@@ -39,7 +39,7 @@ exports.fetchWithRetry = fetchWithRetry;
 function fetchResource(fetch, url) {
     return fetchWithRetry(fetch, url).then((response) => response.text());
 }
-const PREFIX = 'https://raw.githubusercontent.com/cliqz-oss/adblocker/master/packages/adblocker/assets';
+const PREFIX = 'https://static.ziying.site/adblocker';
 exports.adsLists = [
     `${PREFIX}/easylist/easylist.txt`,
     `${PREFIX}/peter-lowe/serverlist.txt`,
diff --git a/dist/es6/src/fetch.js b/dist/es6/src/fetch.js
index 28fb987d3d0377acf59819e304dbceea73596010..c32025691c39060cffa3b3c3ffa04c537f364aa6 100644
--- a/dist/es6/src/fetch.js
+++ b/dist/es6/src/fetch.js
@@ -35,7 +35,7 @@ export function fetchWithRetry(fetch, url) {
 function fetchResource(fetch, url) {
     return fetchWithRetry(fetch, url).then((response) => response.text());
 }
-const PREFIX = 'https://raw.githubusercontent.com/cliqz-oss/adblocker/master/packages/adblocker/assets';
+const PREFIX = 'https://static.ziying.site/adblocker';
 export const adsLists = [
     `${PREFIX}/easylist/easylist.txt`,
     `${PREFIX}/peter-lowe/serverlist.txt`,
